db/migrate/20230407170430_add_payload_credentials_configuration_scripts.rb

```diff
@@ -0,0 +1,14 @@
+class AddPayloadCredentialsConfigurationScripts < ActiveRecord::Migration[6.1]
+  def change
+    change_table :configuration_scripts do |t|
+      t.string     :run_by_userid
+      t.jsonb      :payload
+      t.string     :payload_type
```

Comment:
@Fryguy we talked about having a payload_type so the UI can generically display this